change/@fluentui-react-native-experimental-native-font-metrics-53961c7c-6784-4f8c-b197-79ec8ba74332.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add NativeFontMetrics package",
+  "packageName": "@fluentui-react-native/experimental-native-font-metrics",
```

Comment:
Why is this listed under `experimental`?

Reply:
We tend to prefix packages with `experimental` until they're V1 ready. I personally don't like changing package names so much but 🤷🏾.

For this package in particular, I think it's fair to keep it since we want to eventually use core anyway.

Reply:
As Saad said, we don't plan on this being a permanent thing anyway, so experimental is fine.